src/utils/urls.js

```diff
@@ -52,8 +47,9 @@ export function getUsersUrl() {
   return usersPath;
 }
 
-export function getArticleUrl(slug) {
-  return concatParts(articlesPrefix, slug);
+export function getDocUrl(key, slug) {
+  // We need the check here until we fix the image-tiles component in story https://educandu.atlassian.net/browse/EDU-150
+  return slug ? concatParts(docsPrefix, key, slug) : concatParts(docsPrefix, key);
```

Comment:
Maybe this `concatParts` should be smart enough to first filter, then reduce, so we don't have a temporary hack here.

Reply:
I think that could hide errors if we make concatParts too "smart". This should never be called with null values. What do you think?

Reply:
From my perspective this method should do the full sanitization, as it's already doing slash trimming, it should also remove empty parts.

Reply:
Fixed, and added tests for this.